desktop/apps/consignments2/client/submission.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react'
+import SubmissionFlow from '../components/submission_flow'
+import ThankYou from '../components/thank_you'
+import reducers from './reducers'
+import createHistory from 'history/createBrowserHistory'
+import createLogger from 'redux-logger'
+import thunkMiddleware from 'redux-thunk'
+import { Provider } from 'react-redux'
+import { Router, Route } from 'react-router'
+import { createStore, applyMiddleware } from 'redux'
+import { render } from 'react-dom'
+import { routerMiddleware } from 'react-router-redux'
+
+function setupSubmissionFlow () {
+  const loggerMiddleware = createLogger()
+  const history = createHistory()
+  const store = createStore(
+    reducers,
+    applyMiddleware(
+      thunkMiddleware, // lets us dispatch() functions
+      routerMiddleware(history),
+      loggerMiddleware // middleware that logs actions
+    )
+  )
+
+  render(
+    <Provider store={store}>
+      <Router history={history}>
```

Comment:
I'm not sure if their docs are out of date already, but I could not seem to get this to work with `ConnectedRouter` (and `Router` works okay). Maybe when you are back we can pair on looking at this.